feature/bgp/static_route_bgp_redistribution/otg_tests/static_route_bgp_redistribution_test/static_route_bgp_redistribution_test.go

```diff
@@ -827,6 +866,57 @@ func redistributeIPv4PrefixRoutePolicy(t *testing.T, dut *ondatra.DUTDevice, ate
 	verifyTraffic(t, ate, otgConfig)
 }
 
+func setAttributeViaCLI(t *testing.T, dut *ondatra.DUTDevice, policyName string, statement string, prefixSetName string, setMed int, origin string) {
+	//route-policy route-policy-v4
+	//   #statement-name statement-v4
+	//   if destination in prefix-set-v4 then
+	//     set med 104
+	//     set origin igp
+	//   endif
+	// end-policy
+	cliConfig := fmt.Sprintf("route-policy %s\n #statement-name %v\n if destination in %v then\n", policyName, statement, prefixSetName)
+
+	if setMed != 0 {
+		cliConfig += fmt.Sprintf("  set med %d\n", setMed)
+	}
+	if origin != "" {
+		cliConfig += fmt.Sprintf("  set origin %v\n", origin)
+	}
+	cliConfig += "  done\n endif\nend-policy\n"
+	helpers.GnmiCLIConfig(t, dut, cliConfig)
+}
+
+func setMetricViaCLI(t *testing.T, dut *ondatra.DUTDevice, isV4 bool, metric int, routePolicyName string) {
```

Comment:
Move this functon to a new file in internal/cfgplugins such as `tableconnections.go`

Suggest renaming the function to `DeviationCiscoTableConnectionsBGPtoStaticMetricPropagation` or similar.

Add a description such as:
This function is used as an alternative to  `/network-instances/network-instance/table-connections/table-connection/config/disable-metric-propagation`.  In OC this path is set to 'false' by default, therefore enabling table-connections to propagate metrics from one protocol to another.  This deviation implements CLI to perform the equivalent function.